edge/pkg/edged/edged.go

```diff
@@ -1043,8 +1041,7 @@ func (e *edged) consumePodAddition(namespacedName *types.NamespacedName) error {
 	}
 	curPodStatus, err := e.podCache.GetNewerThan(podUID, t.(time.Time))
 	if err != nil {
-		klog.Errorf("pod %s cache newer failed: %v", podName, err)
-		return err
+		return fmt.Errorf("pod %s cache newer failed: %v", podName, err)
```

Comment:
Seems that the err will not be printed in calling func, please check all the errors modified.